### DISCUSSION TITLE
Is dark mode an in-built feature?

### DISCUSSION THREAD

**Discussion by pastrami1764** [jendrikseipp/rednotebook#698] [Q&A] [answered]
I am the latest Fedora stable release with GNOME with all the default settings. I was expecting the app to automatically adapt to my system-wide dark style but when it didn't, I tried to find a setting to enable dark mode in the settings - which I couldn't find (I tried repeating this with both the distribution package and the Flatpak to no avail). After searching through the help documentation for the app and the internet for this, it seems that dark mode does exist for Windows. I couldn't find a request for this feature in the "Ideas" section of this discussion section either (nor in the "Issues" or "Pull requests").
Is there any way to enable it in Linux?

**Comment by jendrikseipp** [comment]
I thought dark mode would be picked up by the app automatically. Maybe someone else who has successfully turned the app dark can chime in here.

**Comment by curioussushiroll** [comment, marked as answer]
Style in appearance settings of gnome 40+ (which is now used in fedora) only changes the theme for libadwaita (gtk4 only) applications. Meanwhile, rednotebook is gtk3. 

It is possible to change theme for all "legacy" apps using gnome-tweaks => Appearance => Legacy Applications.

If you have already done that and it still doesn't respect dark theme settings, could you please execute "GTK_THEME=Adwaita:dark rednotebook" and tell if it changes the theme to dark?

I apologize if I misunderstood something :P

**Reply by pastrami1764** [reply]
So changing the theme using gnome-tweaks worked (although I did have to switch from using the Flatpak to the Fedora repository version)!
Thanks a lot!

**Comment by ePaulo** [comment]
For those using Windows 10 that prefer dark-mode, add a "settings.ini" file in a "RedNotebook/etc/gtk-3.0" folder, and include the following:
[Settings]
gtk-application-prefer-dark-theme=1

**Reply by HieronymusCH** [reply]
That's not working on Windows 11.
Has anybody a hint for 11?
